DTAConfig/Settings/FileSourceDestinationInfo.cs

```diff
@@ -140,6 +154,13 @@ public void Revert()
                     }
                     break;
 
+                case FileOperationOptions.ImmutableLink:
+                    FileInfo info = new FileInfo (SourcePath);
+                    info.IsReadOnly = false;
```

Comment:
I think disabling readonly is unnecessary since the source file are meant to be readonly